grammar.js

```diff
@@ -455,7 +455,15 @@ module.exports = grammar({
         binaryExpr(prec.left, 4, ">", $._case_clause_guard_expression),
         binaryExpr(prec.left, 4, ">=", $._case_clause_guard_expression),
         binaryExpr(prec.left, 4, ">.", $._case_clause_guard_expression),
-        binaryExpr(prec.left, 4, ">=.", $._case_clause_guard_expression)
+        binaryExpr(prec.left, 4, ">=.", $._case_clause_guard_expression),
+        binaryExpr(prec.left, 5, "+", $._case_clause_guard_expression),
+        binaryExpr(prec.left, 5, "+.", $._case_clause_guard_expression),
+        binaryExpr(prec.left, 5, "-", $._case_clause_guard_expression),
+        binaryExpr(prec.left, 5, "-.", $._case_clause_guard_expression),
+        binaryExpr(prec.left, 5, "*", $._case_clause_guard_expression),
+        binaryExpr(prec.left, 5, "*.", $._case_clause_guard_expression),
+        binaryExpr(prec.left, 5, "/", $._case_clause_guard_expression),
+        binaryExpr(prec.left, 5, "/.", $._case_clause_guard_expression)
```

Comment:
```suggestion
        binaryExpr(prec.left, 6, "*", $._case_clause_guard_expression),
        binaryExpr(prec.left, 6, "*.", $._case_clause_guard_expression),
        binaryExpr(prec.left, 6, "/", $._case_clause_guard_expression),
        binaryExpr(prec.left, 6, "/.", $._case_clause_guard_expression)
```

I believe we want these higher than the `+`/`-`/etc. ones so that the expressions associate properly. For example if you add a branch to one of the `case`s in the test file

```gleam
case value {
  n if n + 1 * 2 > 5 -> False
}
```

without the change it would be `(n + 1) * 2` but it should associate as `n + (1 * 2)`

Reply:
🤦🏻‍♂️ whoops, thanks